internal/api/ui/login/static/i18n/pt.yaml

```diff
@@ -0,0 +1,446 @@
+  Diverse: Diverso / X
+  PasswordLabel: Senha
+  PasswordConfirmLabel: Confirmação de senha
+  TosAndPrivacyLabel: Termos e condições
+  TosConfirm: Eu aceito os
+  TosLinkText: termos de serviço
+  PrivacyConfirm: Eu aceito a
+  PrivacyLinkText: política de privacidade
+  ExternalLogin: ou registre-se com um usuário externo
+  BackButtonText: login
+  NextButtonText: próximo
+
+ExternalRegistrationUserOverview:
+  Title: Registro de usuário externo
+  Description: Coletamos seus detalhes de usuário do provedor selecionado. Agora você pode alterá-los ou completá-los.
+  EmailLabel: E-mail
+  UsernameLabel: Nome de usuário
+  FirstnameLabel: Nome próprio
+  LastnameLabel: Sobrenome
+  NicknameLabel: Apelido
+  PhoneLabel: Número de telefone
+  LanguageLabel: Idioma
+  German: Alemão
+  English: Inglês
+  Italian: Italiano
+  French: Francês
+  Chinese: Chinês simplificado
+  Japanese: Japonês
+  Polish: Polonês
+  Spanish: Espanhol
+  TosAndPrivacyLabel: Termos e condições
+  TosConfirm: Eu aceito os
+  TosLinkText: termos de serviço
+  PrivacyConfirm: Eu aceito a
+  PrivacyLinkText: política de privacidade
+  ExternalLogin: ou registre-se com um usuário externo
+  BackButtonText: voltar
+  NextButtonText: salvar
+
+RegistrationOrg:
+  Title: Registro de organização
+  Description: Insira o nome de sua organização e seus dados de usuário.
+  OrgNameLabel: Nome da organização
+  EmailLabel: E-mail
+  UsernameLabel: Nome de usuário
+  FirstnameLabel: Nome próprio
+  LastnameLabel: Sobrenome
+  PasswordLabel: Senha
+  PasswordConfirmLabel: Confirmação de senha
+  TosAndPrivacyLabel: Termos e condições
+  TosConfirm: Eu aceito os
+  TosLinkText: termos de serviço
+  PrivacyConfirm: Eu aceito a
+  PrivacyLinkText: política de privacidade
+  SaveButtonText: Criar organização
+
+LoginSuccess:
+  Title: Login bem-sucedido
+  AutoRedirectDescription: Você será redirecionado de volta para o seu aplicativo automaticamente. Se isso não acontecer, clique no botão abaixo. Você pode fechar a janela em seguida.
+  RedirectedDescription: Agora você pode fechar esta janela.
+  NextButtonText: próximo
+
+LogoutDone:
+  Title: Logout concluído
+  Description: Você fez logout com sucesso.
+  LoginButtonText: login
+
+LinkingUsersDone:
+  Title: Vinculação de usuários
+  Description: Vinculação de usuários concluída.
+  CancelButtonText: cancelar
+  NextButtonText: próximo
+
+ExternalNotFound:
+  Title: Usuário externo não encontrado
+  Description: Usuário externo não encontrado. Deseja vincular seu usuário ou registrar um novo.
+  LinkButtonText: Vincular
+  AutoRegisterButtonText: registrar
+  TosAndPrivacyLabel: Termos e condições
+  TosConfirm: Eu aceito os
+  TosLinkText: termos de serviço
+  PrivacyConfirm: Eu aceito a
+  PrivacyLinkText: política de privacidade
+  German: Alemão
+  English: Inglês
+  Italian: Italiano
+  French: Francês
+  Chinese: Chinês simplificado
+  Polish: Polonês
+  Japanese: Japonês
+  Spanish: Espanhol
+  Bulgarian: Búlgaro
```

Comment:
Please add portugese here

Reply:
Hi @hifabienne , i've added portuguese. Thanks for the input!